app/views/foods/index.html.erb

```diff
@@ -0,0 +1,63 @@
+<div class="py-6">
+  <div class="max-w-7xl mx-auto px-4 sm:px-6 lg:px-8">
+    <div class="text-center">
+      <h1 class="font-bold text-4xl bg-gray-800 text-white py-2 rounded-md">Foods List</h1>
+      <% if flash[:success] %>
+        <div class="flex p-4 mt-4 text-sm text-green-800 border border-green-300 rounded-lg
+        bg-green-50 dark:bg-gray-800 dark:text-green-400 dark:border-green-800" role="alert">
+          <svg aria-hidden="true"
+          class="flex-shrink-0 inline w-5 h-5 mr-3"
+          fill="currentColor"
+          viewBox="0 0 20 20"
+          xmlns="http://www.w3.org/2000/svg">
+            <path fill-rule="evenodd"
+          d="M18 10a8 8 0 11-16 0 8 8 0 0116 0zm-7-4a1 1 0 11-2 0 1 1 0 012 0zM9 9a1 1 0 000 2v3a1 1 0 001 1h1a1 1 0 100-2v-3a1 1 0 00-1-1H9z" clip-rule="evenodd"></path></svg>
+          <span class="sr-only">Info</span>
+          <div>
+            <span class="font-medium">Success alert!</span> <%= flash[:success] %>
+          </div>
+        </div>
+      <% end %>
```

Comment:
Your notification is beautiful we will be using that later so please add in the components folder so that we can use later(We will be using it in application views)